### PR TITLE
Migrate watcher microservice

### PR DESCRIPTION
**Pull request by maclover7**
This migrates over the `watcher` microservice to `dispatch`. I'd like to try and ditch EventMachine for just bare threads in a followup PR.

cc @andrew

### CODE REVIEW

File: dispatch.rb [hunk 1]
```diff
@@ -90,9 +94,130 @@ def send_event(name, params = {})
...
+    ['http://npm-update-stream.libraries.io/', 'NPM'],
+    ['https://rubygems.org/api/v1/activity/just_updated.json', 'Rubygems'],
+    ['https://rubygems.org/api/v1/activity/latest.json', 'Rubygems'],
+    ['https://atom.io/api/packages?page=1&sort=created_at&direction=desc', 'Atom'],
+    ['https://atom.io/api/packages?page=1&sort=updated_at&direction=desc', 'Atom'],
+    ['http://package.elm-lang.org/new-packages', 'Elm'],
+    ['https://crates.io/summary', 'Cargo'],
+    ['http://api.metacpan.org/v0/release/_search?q=status:latest&fields=distribution&sort=date:desc&size=100', 'CPAN'],
+    #['https://hex.pm/api/packages?sort=inserted_at', 'Hex'],
+    #['https://hex.pm/api/packages?sort=updated_at', 'Hex']
```
Comment (andrew) [inline dispatch.rb]: I'm guessing you hit the rate limit on the Hex API?

Comment (maclover7) [inline dispatch.rb]: Yeah, weird things were happening on these endpoints. Assuming I should uncomment?

Comment (andrew) [inline dispatch.rb]: Yes please

**Comment by maclover7** [comment]
@andrew updated

**Comment by andrew** [comment]
🎉  Hoping to get this deployed next week

**Comment by maclover7** [comment]
👍 👍

**Comment by andrew** [comment]
This is now live in production, I've turned off the watcher service 🎉

**Comment by maclover7** [comment]
👍 👍 can you regenerate the infrastructure pic in librariesio/documentation?

**Comment by andrew** [comment]
@maclover7 done! https://github.com/librariesio/documentation/commit/87d219d9de2ce37cd74980a2f5dbd6730e16b2df

**Pull request merged**